package.json

```diff
@@ -27,37 +27,52 @@
   "devDependencies": {
     "@babel/core": "^7.6.0",
     "@babel/preset-env": "^7.6.0",
+    "@commitlint/cli": "^8.2.0",
+    "@commitlint/config-conventional": "^8.2.0",
+    "@semantic-release/changelog": "^3.0.4",
+    "@semantic-release/git": "^7.0.16",
     "@webex/eslint-config-base": "^0.8.0",
     "babel-eslint": "^10.0.3",
+    "conventional-changelog-angular": "^5.0.3",
     "eslint": "^6.3.0",
     "eslint-config-prettier": "^6.2.0",
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-mocha": "^6.1.0",
     "eslint-plugin-prettier": "^3.1.0",
-    "jest": "^24.9.0",
```

Comment:
I think you merged things incorrectly, we still need Jest and Jest configurations